Documentation/op-guide/performance.md

```diff
@@ -0,0 +1,13 @@
+# Performance
+
+## Understanding performance
+
+etcd provides stable, sustained high performance. Two factors define performance: latency and throughput. Latency is the time taken to complete an operation. Throughput is the total operations completed within some time period. Usually average latency increases as the overall throughput increases when etcd accepts concurrent client requests. On a commodity cloud environment (e.g., standard n-4 on GCE or a comparable machine type on AWS), a three member etcd cluster finishes a request in less than one millisecond under light load, and can complete more than 30,000 requests per second under heavy load.
```

Comment:
"In common cloud environments, like a standard `n-4` on Google Compute Engine (GCE) or a comparable machine type on AWS, ..."